RELEASES.md

```diff
@@ -21,6 +21,38 @@ To summarize: **All our ibc-go releases allow chains to communicate successfully
 
 We ensure all major releases are supported by relayers ([hermes](https://github.com/informalsystems/ibc-rs), [rly](https://github.com/strangelove-ventures/relayer) and [ts-relayer](https://github.com/confio/ts-relayer) at the moment) which can relay between the new major release and older releases. We have no plans of upgrading to an IBC protocol specification v2.0, as this would be very disruptive to the ecosystem.
 
+## Release cycle
+
+IBC-Go follows a traditional release cycle involving an alpha, beta, and rc (release candidate) releases before finalizing a new version. As ibc-go works in a non-traditional area, we apply our own interpretation to each release type. We reserve the right to make both go API breaking changes and state machine breaking changes throughout the entire release cycle. The stable release guarentees do not go into affect until a final release is performed. 
+
+It is never advisable to use a non-final release in production. 
+
+### Alpha
+
+Alpha releases are intended to make avaliable new features as soon as they are functional. No correctness guarentees are made and alpha releases MAY contain serious security vulnerabilities, bugs, and lack of user tooling. Alpha releases may be made with known bugs and security vulnerabilities (so long as they don't affect the core functionality).
+
+Initial users of alpha releases are expected to be advanced, patient, and capable of handling unusal errors. Very basic integration testing will be performed by the ibc-go development team before alpha releases.  
+
+An internal audit is typically performed before the alpha release allowing the development team to guage the maturity and stability of changes included in the next release. 
+
+### Beta
+
+Beta releases are intended to signal design stability. While the go API is still subject to change, the core design of the new features should not be. Developers integrating the new features should expect to handle breaking changes when upgrading to RC's. 
+
+Beta releases should not be made with known bugs or security vulnerabilities. Beta releases should focus on ironing out remaining bugs and filling out the UX functionality required by a final release. Beta releases should have a clearly defined scope of the features that will be included in the release. Only highly requested feature additions should be acted upon in this phase. 
+
+When the development team has determined a release is ready to enter the RC phase, a final security audit should be performed. The security audit should be limited to looking for bugs and security vulnerabilities. Code improvements may be noted, but they should not be acted upon unless highly desirable. 
+
+### RC 
+
+RC's are release candidates. Final releases should contain little to no changes in comparison to the latest RC. Changes included in between RC releases should be limited to:
+- improved testing
+- UX additions
+- bug fixes
+- highly requested changes by the community
```

Comment:
Minor nit: Should we maybe capitalise the start of each bullet. E.g. "Improved testing", "Bug fixes"